BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -7,14 +7,14 @@ import Foundation
 /// This struct defines all the standard image identifiers of icons and images used in the app.
 /// When adding new identifiers, please respect alphabetical order.
 /// Sing the song if you must.
-public struct StandardImageIdentifiers {
+public enum StandardImageIdentifiers {
```

Comment:
LGTM i have to refactor the Acorn script then since i'm generating structs there

Reply:
wait wait. nonoono

Reply:
This was a mistake. hold on.

Reply:
Ok. I fixed things. I was testing some swiftformat stuff last night and forgot to disable it so it make changes.

Reply:
No problem @adudenamedruby 💪